torch/_dynamo/variables/misc.py

```diff
@@ -635,6 +635,29 @@ def call_method(
         return super().call_method(tx, name, args, kwargs)
 
 
+class MethodWrapperVariable(variables.ConstantVariable):
+    def call_function(
+        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
+    ) -> "VariableTracker":
+        if self.value.__name__ == "__get__":
+            assert len(args) == 1 and len(kwargs) == 0
+            return args[0].var_getattr(tx, self.value.__self__.__name__)
```

Comment:
This isn't really an accurate implementation

```
>>> torch.Tensor.dtype.__get__(object())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: descriptor 'dtype' for 'torch._C._TensorBase' objects doesn't apply to a 'object' object
```

Reply:
this is a good catch, I'll add a check that it's the expected type